mne/epochs.py

```diff
@@ -879,6 +879,10 @@ def drop_log_stats(self, ignore=['IGNORED']):
         -------
         perc : float
             Total percentage of epochs dropped.
+
+        See Also
+        --------
+        plot_drop_log
```

Comment:
that's intentional. See this: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt